### ISSUE TITLE
Old sync is going to be removed from FF

### ISSUE THREAD

**Issue by gerroon** [owncloud/mozilla_sync#153]
Hi

It looks like the old sync will be removed from FF with 45 or 46. An that means that the mobile version wont have it also.

I am just reporting it because maybe the devs can add the info about the latest version that this extension supports in the wiki and readme.

https://bugzilla.mozilla.org/show_bug.cgi?id=1220906

**Comment by jancborchardt** [comment]
Maybe it also makes sense to move this app to https://github.com/owncloudarchive to make clear that it’s old technology which is not supported anymore.

@ogasser would you be ok with this?

cc @karlitschek @DeepDiver1975

**Comment by gerroon** [comment]
I personally do not think that that is necessary really since it is not the oc app that is the old tech it is just that FF is removing the support. As long as the devs work on it it should stay in the current branch I think.

I am personally not going to upgrade my FF until I find a solution to account sync.

**Comment by jancborchardt** [comment]
> As long as the devs work on it it

That’s the other issue though. ;)

**Comment by ogasser** [comment]
I agree with @jancborchardt to move this app to the archive. Unfortunately I don't have time to work on an ownCloud implementation of the new Firefox sync. Additionally, it isn't even possible to input a custom sync server on Firefox on Android.

**Comment by htgoebel** [comment]
So at least you should state in the README that this app is only for the old Mozilla sync and will not work with the new one. This will save people wasting time for investigating this fact - or even wasting more time for trying it out. Thanks!

**Comment by ghost** [comment]
I really like this feature as I have my own OC server and don't know what will happen when even Firefox 45 goes EOL.

Firefox 45 ESR and earlier versions of Iceweasel 44(Firefox came back to Debian, thus replaced Iceweasel) work on Debian Testing with moz_sync. Firefox 45 will be in Debian Stable(9/stretch) next year as far as I can imagine.

Both Debian Wheezy and Jessie supports Iceweasel/FF 38.

If anyone's interested, newer versions of FF/Iceweasel on stable releases are available at http://mozilla.debian.net/.

**Comment by ghost** [comment]
Also a dup of https://github.com/owncloud/mozilla_sync/issues/33 where plenty of people already added some background info and stuff.

**Comment by ghost** [comment]
Just wanted to say owncloud_sync still works on Firefox 46. It nicely retrieves preferences, add-ons and bookmarks from owncloud 8.1.something(production branch).

Tested on a Freebsd 10.2 VM.

**Comment by gerroon** [comment]
@richardskumat 

Weird it is disabled on FF 45 mobile

**Comment by ghost** [comment]
@gerroon 

Firefox 45 via Fdroid

Samsung S3 mini, android 4.12

kernel 3.0.31

It doesn't work for me either on Android 4.1.2, most likely due to SNI and my self-signed certificate.

I went to settings > Accounts > Add Accounts > Firefox Sync (Deprecated) but then I get:

Invalid server url errors.

I've watched my web server url's log and it doesn't even try to connect to owncloud.

My url I've tried:

https://sub.domain.com/owncloud/remote.php/mozilla_sync/

I've gone ahead and made the 2 about:config(services.sync.tokenServerURI,services.sync.username) changes on https://github.com/owncloudarchive/mozilla_sync/blob/master/INSTALL.md.

Now the settings part with deprecated firefox account seems to be unresponsive, not responding to any touches.

Edit:

On desktops and VM-s, Firefox 45, 38 and even 46 seems to still work.
